src/plugins/wizard/public/embeddable/wizard_embeddable.tsx

```diff
@@ -22,44 +22,15 @@ import { DataPublicPluginStart } from '../../../data/public';
 
 export const WIZARD_EMBEDDABLE = 'WIZARD_EMBEDDABLE';
 
-// TODO: remove search, hasMatch or update as appropriate
-export interface WizardInput extends SavedObjectEmbeddableInput {
-  /**
-   * Optional search string which will be used to highlight search terms as
-   * well as calculate `output.hasMatch`.
-   */
-  search?: string;
-}
-
 export interface WizardOutput extends EmbeddableOutput {
-  /**
-   * Should be true if input.search is defined and the task or title contain
-   * search as a substring.
-   */
-  hasMatch: boolean;
   /**
    * Will contain the saved object attributes of the Wizard Saved Object that matches
    * `input.savedObjectId`. If the id is invalid, this may be undefined.
    */
   savedAttributes?: WizardSavedObjectAttributes;
 }
 
-/**
- * Returns whether any attributes contain the search string.  If search is empty, true is returned. If
- * there are no savedAttributes, false is returned.
- * @param search - the search string
- * @param savedAttributes - the saved object attributes for the saved object with id `input.savedObjectId`
- */
-function getHasMatch(search?: string, savedAttributes?: WizardSavedObjectAttributes): boolean {
```

Comment:
Do we know why we are removing this? or are we just taking a guess that we dont need it? Not knowing much about what this does, it looks like the code that decides if the search query in add panel matches the embeddable. But thats just my hypothesis.

Reply:
This isn't necessary - it was part of the example embeddable from `todo_ref_embeddable`, which includes a search/highlight text feature, but mostly to demonstrate embeddable input/output. There's nothing like it in vizualizations embeddables.